src/manage_sql_nvts.c

```diff
@@ -1700,7 +1738,8 @@ update_nvt_cache_osp (const gchar *update_socket, gchar *db_feed_version,
  *
  * @param[in]  update_socket  Socket to use to contact ospd-openvas scanner.
  *
- * @return 0 success, -1 error, 2 scanner still loading.
+ * @return 0 success, -1 error, 1 VT integrity check failed,
+ *         2 scanner still loading.
```

Comment:
And same here.

Reply:
I know manage_update_nvt_cache_osp doc has this return but it's not true there either, and I don't see it happening anytime soon, so I don't think we should spread that.

Reply:
I think @jjnicola recently changed OSPd(-OpenVAS) to return an error if it is still loading
but it would be cleaner if we handled this in a separate PR that also considers other uses of OSP.

Reply:
Ah, OK so they could have stayed.  Thanks.